docs/abm.sql

```diff
@@ -1,152 +0,0 @@
-CREATE TABLE `build_result` (
```

Comment:
Why remove this file?

Reply:
Not sure why this is removed.

Reply:
I meant: this file should not be removed. Please add it again.